docs/log-correlation.asciidoc

```diff
@@ -33,16 +33,20 @@ See the relevant https://www.elastic.co/guide/en/apm/agent/index.html[Agent docu
 [float]
 ==== Add APM identifiers to your logs
 
-Once log correlation is enabled,
-you must ensure your logs contain APM identifiers.
+Once log correlation is enabled, you must ensure your logs contain APM identifiers.
+
+Log correlation relies on two set of fields:
```

Comment:
```suggestion
Log correlation relies on these fields:
```
For future proofing, it's best not to mention number of items because the number may change in the future. (It's pretty common for folks to overlook intros like this when making updates.)